### PR TITLE
[10.0][MIG] account_credit_control_dunning_fees

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends : 
- [x] https://github.com/OCA/account-financial-tools/pull/462

### CODE REVIEW

**Comment by ovnicraft** [comment]
#462 was merged, anything else to merge this  ?

**Comment by grindtildeath** [comment]
@ovnicraft I suppose it needs a review first.

File: account_credit_control_dunning_fees/__manifest__.py [hunk 1]
```diff
@@ -33,7 +14,7 @@
           ],
  'demo': [],
```
Comment (simahawk) [inline account_credit_control_dunning_fees/__manifest__.py]: remove emtpy keys

File: account_credit_control_dunning_fees/model/dunning.py [hunk 2]
```diff
@@ -1,24 +1,7 @@
...
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
-from openerp import models, api
+# Copyright 2017
```
Comment (simahawk) [inline account_credit_control_dunning_fees/model/dunning.py]: copyright who? :)

**Review by simahawk — APPROVED** [review]
LGTM, just 2 small remarks

**Review by simahawk — APPROVED** [review]

**Review by guewen — APPROVED** [review]

**Pull request merged**